### PR TITLE
Binary Patient Access Updated

### PR DESCRIPTION
**Pull request by NimishaMN**
Description
----
Binary Patient Access Updated
Screenshot 2021-03-01 at 7 28 55 PM
Screenshot 2021-03-01 at 7 29 11 PM
 
PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/other/binary.md [hunk 1]
```diff
@@ -61,6 +61,21 @@ The `Binary.read` scope and either the `DiagnosticReport.read` or the `DocumentR
 
 <%= disclaimer %>
 
+#### Patient Authorization Request
+
+  ### Example
+
+  #### Request
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Binary/XR-197198634
```
Comment (aki-mathur) [inline content/millennium/r4/other/binary.md:70]: For Patient Access, it should be closed endpoint
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Binary/XR-197198634

File: content/millennium/r4/other/binary.md [hunk 2]
```diff
@@ -102,6 +117,21 @@ An `Accept` header of `application/xml` could be supplied instead, if the XML Bi
 
 <%= disclaimer %>
 
+#### Patient Authorization Request
+
+  ### Example
+
+  #### Request
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Binary/$autogen-ccd-if?patient=1694010
```
Comment (aki-mathur) [inline content/millennium/r4/other/binary.md:126]: Update it to closed end points 
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Binary/XR-197198634

**Review by aki-mathur — CHANGES_REQUESTED** [review]
I guess you need to update the binary_r4_examples file also to have closed endpints

File: content/millennium/r4/other/binary.md (hunk 1 above)
Comment (NimishaMN) [reply content/millennium/r4/other/binary.md]: Updated https://github.com/cerner/fhir.cerner.com/pull/563/commits/ab6969b15631293ec930c2aeffa90629c3eeb1ca

File: content/millennium/r4/other/binary.md (hunk 2 above)
Comment (NimishaMN) [reply content/millennium/r4/other/binary.md]: Updated https://github.com/cerner/fhir.cerner.com/pull/563/commits/ab6969b15631293ec930c2aeffa90629c3eeb1ca

**Review by shubhamvinayak — APPROVED** [review]

**Review by aki-mathur — APPROVED** [review]

**Comment by MaxPhilips** [comment]
live
Screen Shot 2021-03-05 at 12 51 45 PM

**Pull request merged**